velox/dwio/dwrf/test/TestDecompression.cpp

```diff
@@ -557,7 +557,8 @@ TEST_F(DecompressionTest, testSmallBufferInflate) {
       0xe, 0x0, 0x0, 0x63, 0x60, 0x64, 0x62, 0xc0, 0x8d, 0x0};
   const std::unique_ptr<SeekableInputStream> result = createTestDecompressor(
       CompressionKind_ZLIB,
-      std::make_unique<SeekableArrayInputStream>(buffer, std::size(buffer)),
+      std::make_unique<SeekableArrayInputStream>(
+          buffer, VELOX_ARRAY_SIZE(buffer)),
```

Comment:
It seems better to use std::size, replace existing instances of VELOX_ARRAY_SIZE and delete that macro. Am I missing something?